docs/installing-solidity.rst

```diff
@@ -362,7 +362,7 @@ And for Windows:
 
     mkdir build
     cd build
-    cmake -G "Visual Studio 15 2017 Win64" ..
+    cmake -G "Visual Studio 16 2019 Win64" ..
```

Comment:
Maybe we should also mention here that, if this is supposed to use the boost version built by ``install_deps.ps1``, it also needs ``-DCMAKE_MSVC_RUNTIME_LIBRARY=MultiThreaded`` and ``-DBoost_DIR="..\deps\boost\lib\cmake\Boost-<BoostVersion>``?

Reply:
(Which is the complications I warned about in https://github.com/ethereum/solidity/pull/9811#discussion_r489353372 ;-))

Reply:
Can you do that change, please?